tests/wasm/README.md

```diff
@@ -9,14 +9,11 @@ at the top, and any other int-to-int functions below.
 Run:
 
 ```shell
-clang --target=wasm32 -mmultivalue -Xclang -target-abi -Xclang experimental-mv --no-standard-libraries -Wl,--export-all -Wl,--no-entry -o <filename>.wasm <filename>.c
+clang --target=wasm32 -mcollatz.c -Xclang -target-abi -Xclang experimental-mv --no-standard-libraries -Wl,--export-all -Wl,--no-entry -o <filename>.wasm <filename>.c
```

Comment:
```suggestion
clang --target=wasm32 -Xclang -target-abi -Xclang experimental-mv --no-standard-libraries -Wl,--export-all -Wl,--no-entry -o <filename>.wasm <filename>.c
```

Reply:
I thought this was the filename as well... Thank you for spotting!